SMARTAGRIFOOD/vineyardData/schema.json

```diff
@@ -1,165 +1,227 @@
+            "format": "uri",
+            "type": "string",
+          },
+          "alternateName": {
+            "description": "Property. An alternative name for this item",
+            "type": "string"
+          },
+          "dataProvider": {
+            "description": "Property. A sequence of characters identifying the provider of the harmonised data entity.",
+            "type": "string"
+          },
+          "dateCreated": {
+            "description": "Property. Entity creation timestamp. This will usually be allocated by the storage platform.",
+            "format": "date-time",
+            "type": "string"
+          },
+          "dateModified": {
+            "description": "Property. Timestamp of the last modification of the entity. This will usually be allocated by the storage platform.",
+            "format": "date-time",
+            "type": "string"
+          },
+          "description": {
+            "description": "Property. A description of this item",
+            "type": "string"
+          },
+          "id": {
+            "description": "Property. Unique identifier of the entity"
+          },
+          "isOrganic": {
+            "description": "Property. True if vineyard zone is treated with organic products only, otherwise false",
+            "type": "boolean"
+          },
+          "name": {
+            "description": "Property. The name of this vineyard block.",
+            "type": "string"
+          },
+          "size": {
+            "description": "Property. Size of this vineyard block in square meters.",
+            "type": "number"
+          },
+          "numVinesRepresented": {
+            "description": "Property. Number of vines in this vineyard block.",
+            "type": "integer"
+          },
+          "alignment": {
+            "description": "Property. Direction of vineyard block’s rows if vineyard block is planted in rows",
+            "enum": [
+              "N-S",
+              "NE-SW",
+              "E-W",
+              "SE-NW"
+            ],
+            "type": "string"
+          },
+          "inclination": {
+              "aspect": {
+                "description": "‘Property. Aspect of the inclination’",
+                "enum": [
+                  "N",
+                  "NE",
+                  "E",
+                  "SE",
+                  "S",
+                  "SW",
+                  "W",
+                  "NW"
+                ],
+                "type": "string"
+              },
+              "slope": {
+                "description": "‘Property. Slope in degrees’",
+                "type": "string"
+              }
+          },
+          "plants": {
+            "trainingSystem": {
+              "description": "Property. Vine training system.",
+              "type": "string"
+            },
+            "variety": {
+              "description": "Vine variety.",

```

Comment:
need to define if it is a property/relationship/geoproperty. 
it would be good to use this [checking service](https://smartdatamodels.org/index.php/data-models-contribution-api/
)